### DISCUSSION TITLE
Latest setup

### DISCUSSION THREAD

**Discussion by PitToad** [rsjaffe/MIDI2LR#722] [Show and tell]
2 x Dell S2721QS monitors
Logi M570 trackball with front/back buttons remapped to white balance/crop
Midifighter Twister and 3D for main editing.

DIY riser desk for editors made of 15mm ply and some timber with kitchen-cupboard door handles and bathroom mirror standoffs to stop the decks moving around.
Custom USB leads cos I got really bored one day

Switchable USB hub for no apparent reason :-)

![20221229_200225](https://user-images.githubusercontent.com/111119318/210006669-9ae3e9d0-3261-4f89-8e2f-719475d7fa1e.jpg)

![20221229_200239](https://user-images.githubusercontent.com/111119318/210006712-dd83cd82-3976-469c-b567-3351e5e8d2f0.jpg)

![20221229_200253](https://user-images.githubusercontent.com/111119318/210006719-05f52ca2-5871-4d0d-a66a-b5e151688a89.jpg)

![20221229_200851](https://user-images.githubusercontent.com/111119318/210006721-775fc334-08fd-49fc-9498-865123fe5b3c.jpg)

![20221229_201020](https://user-images.githubusercontent.com/111119318/210006722-ae5ddd8f-62cc-45fe-9e5e-0de30cae544f.jpg)

![20221229_201029](https://user-images.githubusercontent.com/111119318/210006723-e1ecb3e2-50fa-414c-becf-065bb2ba9655.jpg)

**Comment by cstsang** [comment]
Beautiful setup!
